data/update/mysql/dbupdate-7.5.latest-to-8.0.0.sql

```diff
@@ -3,3 +3,13 @@ UPDATE ezsite_data SET value='8.0.0' WHERE name='ezpublish-version';
 ALTER TABLE ezcontentclass_attribute MODIFY data_text1 VARCHAR(255);
 
 ALTER TABLE ezcontentclass_attribute ADD COLUMN is_thumbnail TINYINT(1) NOT NULL DEFAULT '0';
+
+--
+-- EZP-31471: Keywords versioning
+--
+
+ALTER TABLE `ezkeyword_attribute_link` ADD COLUMN `versions` TEXT NULL DEFAULT NULL;
```

Comment:
Not to sound too negative, but I am not sure that adding a varchar column with the list of versions that reference this keyword is the best solution.
I am thinking esp. about other existing field types that implement external storage, such as ezimage or ezfile: it would be good if they all used the same strategy to 'map' the external 'asset' to multiple obj versions...

Reply:
I am also not fan of storing versions serialized.

Reply:
@gggeek @ViniTou My reasoning to use serialized versions column was that I originally started developing this solution for version 2.5, so I kinda wanted to keep old database layout intact. I'm gonna change this though, thanks for the feedback.